packages/web/src/components/card/Card.tsx

```diff
@@ -49,10 +56,13 @@ export type CardContentProps = FlexProps
 
 export const CardContent = (props: CardContentProps) => {
   const { size } = useContext(CardContext)
+  const { spacing } = useTheme()
+  const cardWidth = cardSizes[size]
 
   return (
     <Flex
       direction='column'
+      w={cardWidth ? cardWidth - spacing.s * 2 : undefined}
```

Comment:
a bit susss... i guess it's subtracting horizontal padding from desired cardWidth or something?

Reply:
yeah really sus... having exact width is the only way to get ellipses working consistently, will add a comment

Reply:
oh wait im tripping, improvement incoming